### PR TITLE
add script to start scala notebook

### PR DESCRIPTION
**Pull request by xuex2017**

### CODE REVIEW

File: scripts/jupyter-with-zoo-scala.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    echo "Please set SPARK_HOME environment variable"
+    exit 1
+fi
+
+# setup paths
+export ANALYTICS_ZOO_JAR=`find ${ANALYTICS_ZOO_HOME}/lib -type f -name "analytics-zoo*jar-with-dependencies.jar"`
+export ANALYTICS_ZOO_CONF=${ANALYTICS_ZOO_HOME}/conf/spark-analytics-zoo.conf
+
+# Check files
+if [ ! -f ${ANALYTICS_ZOO_CONF} ]; then
```
[the remark this answers is not in the document]
Comment (xuex2017) [reply scripts/jupyter-with-zoo-scala.sh]: no. we can delete these lines

File: scripts/jupyter-with-zoo-scala.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    exit 1
+fi
+
+if [ -z "${SPARK_HOME}" ]; then
+    echo "Please set SPARK_HOME environment variable"
+    exit 1
+fi
+
+# setup paths
+export ANALYTICS_ZOO_JAR=`find ${ANALYTICS_ZOO_HOME}/lib -type f -name "analytics-zoo*jar-with-dependencies.jar"`
```
Comment (hkvision) [inline scripts/jupyter-with-zoo-scala.sh]: Wondering about this: as you define ANALYTICS_ZOO_JAR in this script, and you use ANALYTICS_ZOO_JAR in $1, will this cause a recursion or deadlock?...

Comment (xuex2017) [reply scripts/jupyter-with-zoo-scala.sh]: I have tested this script and the notebook works fine.

Comment (xuex2017) [reply scripts/jupyter-with-zoo-scala.sh]: I've tested the script and the notebook works fine. shouldn't be a problem..

Comment (hkvision) [reply scripts/jupyter-with-zoo-scala.sh]: Emmm...even though it works fine, won't it confuse the users? as they don't even know or realize what ANALYTICS_ZOO_JAR is... Pending this for discussion with other colleagues...

**Comment by hkvision** [comment]
Maybe can you try the following?
put `${ANALYTICS_ZOO_JAR}` inside `jupyter-with-zoo-scala.sh` only and let users specify other options passing as parameters?

**Comment by xuex2017** [comment]
updated jupyter-with-zoo-scala.sh

**Comment by hkvision** [comment]
Seems that you are including other's commits in this PR? See files changed...

**Comment by xuex2017** [comment]
fixed

**Pull request closed without merging**